operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -81,12 +81,13 @@ We follow [the Agile Release Train method](https://www.scaledagileframework.com/
 For P0 bugs (eg. bugs that affect more than 25% of our customers with system degradation), we will do an exception and rollout a hotfix as soon as possible.
 
 **Schedule for Cloud releases**:
- - Thursdays: Merge master into the cloud branch and update cloud test servers.
- - Friday-Tuesday: Release testing and bug fixing.
-    - QA approval should be given by the EOD Tuesday, so that the release rollout can be started early Wednesday morning. 
-    - If QA approval is not ready by EOD Tuesday due to a testing delay or due to a last minute high priority bug fix, then we miss the release train.
+ - Thursdays: Release day. Also merge master into the cloud branch and update cloud test servers, including Rainforest RFQA-Cloud servers.
```

Comment:
```suggestion
 - Thursdays: Release day. Also merge master into the cloud branch and update cloud test servers for the next release, including Rainforest RFQA-Cloud servers.
```

Reply:
Oops! Sorry Amy, I think I messed up and committed your suggestion in error? Apologies - I don't know how to undo it?

Reply:
You did it correctly :) Normally the process is for the PR submitter to commit reviewer suggestions.

Reply:
Aah, pshew! :)  Thank goodness...  Been a while so I'm a bit rusty on PR processes... Thanks Amy